### PR TITLE
Add a buildkite verify pipeline to inspec-aws

### PR DESCRIPTION
**Pull request by miah**
Hello =)

This PR adds expeditor, and a buildkite verification pipeline to inspec-aws.

With expeditor, we get a managed release process on this project. This means we get version numbers for releases, and can easily manage buildkite pipelines!

With buildkite we can start doing integration testing between InSpec and inspec-aws! I'll have another PR that follows this that adds our integration testing. I'm still working with releng on how this looks.

My goal with all of this work is to improve integration testing between InSpec and our resource packs and other products like Automate.

*edit

This will also add /expeditor commands on slack to handle the release process. We'll add another PR with documentation on this =)

### CODE REVIEW

File: .expeditor/config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ 
+changelog:
+ categories:
+  - "Type: New Resource": "New Resources"
+  - "Type: New Feature": "New Features"
+  - "Type: Enhancement": "Enhancements"
+  - "Type: Bug": "Bug Fixes"
+
+merge_actions:
+ - built_in:bump_version:
```
Comment (rmoles) [inline .expeditor/config.yml]: This all looks good! I was just curious about the version bump. Does expeditor's `bump_version` require the `VERSION` file and then we use that to populate the Inspec yaml?

Comment (scotthain) [reply .expeditor/config.yml]: The `bump_version` requires a version file, which is customizeable. https://expeditor.chef.io/docs/reference/built_in/#bump-version for more info

File: .expeditor/update_version.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# After a PR merge, Chef Expeditor will bump the PATCH version in the VERSION file.
+# It then executes this file to update any other files/components with that new version.
+#
+
+set -evx
```
Comment (scotthain) [inline .expeditor/update_version.sh]: Still want that `-x` there?

Comment (miah) [reply .expeditor/update_version.sh]: Good call I dropped it =)

File: .expeditor/config.yml (hunk 1 above)
Comment (miah) [reply .expeditor/config.yml]: Right and bump_version fires off .expeditor/update_version.sh through the only_if.

**Pull request merged**